### PR TITLE
Rf cep26

### PR DESCRIPTION
**Pull request by FlanFlanagan**

### CODE REVIEW

File: source/cep/cep26.rst [hunk 1]
```diff
@@ -84,12 +84,51 @@ rejected.
...
 Specification \& Implementation
 ===============================
-
+To accomplish the methodology proposed here will require some changes to the API within 
+the dynamic resource exchance and the cyclus core code. 
+
+1. Bids will need to be able to hold a unit cost. The API will need to support developers 
+   accessing and setting this cost. 
+2. Update Requests to contain a max-unit-cost instead of a preference. 
+3. The current greedy solver will need to be updated or replaced to accomidate the
```
Comment (scopatz) [inline source/cep/cep26.rst]: "accomodate"

File: source/cep/cep26.rst [hunk 2]
```diff
@@ -84,12 +84,51 @@ rejected.
...
+   support this change. 
+
+The first change will be to add the ability for bids to hold a unit cost value. The 
+implimentation of this will be simple as it will mirror the implimentation of the 
+preference attribute of requests. Therefore all of the techniques used there can be 
+once again used here. Going foward the request max cost will still be the default 
+cost for a request-bid arc. 
+
+Additionally the change from preference to unit cost on the request is primarily a 
+nominclature change. Therefore this update will be simple. The majority of the
```
Comment (scopatz) [inline source/cep/cep26.rst]: "nomenclature", I think

File: source/cep/cep26.rst [hunk 3]
```diff
@@ -84,12 +84,51 @@ rejected.
...
+once again used here. Going foward the request max cost will still be the default 
+cost for a request-bid arc. 
+
+Additionally the change from preference to unit cost on the request is primarily a 
+nominclature change. Therefore this update will be simple. The majority of the 
+work required will be updating all calls of this function currently in use 
+throughout the many archetypes and cyclus core code.  
+
+Once Bids and Requests have their own unit costs, updating the default solver for cyclus 
+will be done to perform a true global optimization of the entire trade system each
```
Comment (scopatz) [inline source/cep/cep26.rst]: no need for the word true here

File: source/cep/cep26.rst [hunk 4]
```diff
@@ -84,12 +84,51 @@ rejected.
...
+cost for a request-bid arc. 
+
+Additionally the change from preference to unit cost on the request is primarily a 
+nominclature change. Therefore this update will be simple. The majority of the 
+work required will be updating all calls of this function currently in use 
+throughout the many archetypes and cyclus core code.  
+
+Once Bids and Requests have their own unit costs, updating the default solver for cyclus 
+will be done to perform a true global optimization of the entire trade system each 
+timestep. This can be done by collecting all of the possible request-bid pairs.
```
Comment (scopatz) [inline source/cep/cep26.rst:111]: We call them arcs or request-bid arcs

File: source/cep/cep26.rst [hunk 5]
```diff
@@ -84,12 +84,51 @@ rejected.
...
+
+Once the pairs have been created, the solver can sort the value of their unit cost 
+from smallest to largest, therefore minimizing the total cost of the system.
+
+This change represents a fundamental change to the behavior of the cyclus simulator. As 
+mentioned there will be several changed to the cyclus core code due to this change. We 
+will aimed to update all of these locations with the new code as well as documentation 
+to help developers update their software and to support future developers using Cyclus. 
+
+It is our aim that this change function as a staged point for a Cyclus 2.0 release.
```
Comment (scopatz) [inline source/cep/cep26.rst]: I think this line could be removed in favor of the below

Comment (scopatz) [reply source/cep/cep26.rst]: moving this to the following section that is

**Comment by FlanFlanagan** [comment]
ping @scopatz

**Pull request merged**